RecoLocalTracker/SiPixelClusterizer/test/gpuClustering_t.h

```diff
@@ -31,7 +31,7 @@ int main(void) {
   using pixelTopology::Phase1;
 
   constexpr int numElements = 256 * maxNumModules;
-  constexpr SiPixelClusterThresholds clusterThresholds(kSiPixelClusterThresholdsDefaultPhase1);
+  const SiPixelClusterThresholds clusterThresholds(2000, 4000, 0.f, 0.f, 0.f, 0.f);
```

Comment:
Can the use of a named initialiser be kept, instead of going back to "magic numbers" ?